lib/cachex/services/courier.ex

```diff
@@ -76,7 +76,13 @@ defmodule Cachex.Services.Courier do
               rescue
                 e -> { :error, Exception.message(e) }
               end
-            send(parent, { :notify, key, result })
+            # Commit the result immediately to avoid a race in fetch
+            # between Get.execute and Courier.dispatch.
+            normalized = normalize_commit(result)
```

Comment:
For this block, please keep the same spacing from before:

```
normalized = normalize_commit(result)

with { :commit, val } <- normalized do
  Put.execute(cache, key, val, const(:notify_false))
end

send(parent, { :notify, key, normalized })
```

Reply:
OK, will do